### DISCUSSION TITLE
Automation response/output passed to Trigger Automation Action

### DISCUSSION THREAD

**Discussion by manwithpowers** [Budibase/budibase#10444] [Help] [answered]
Is there anyway to have the response/output of an automation passed back to the Trigger Automation action that triggered it? Specifically, I have an automation that makes a webhook call (via Integromat automation step), which is triggered by a Trigger Automation action via a button in my app. The server receiving the webhook call sends along a response and I don't see an obvious way of surfacing that back to my app (in this case I would like to use it in a custom notification).
Thanks in advance for the help!

**Comment by ConorWebb96** [comment, marked as answer]
Hey @manwithpowers,

It isn't possible to return the response of an automation back to the client due to it being done asynchronously. I have however added an enhancement which can be found here: https://github.com/Budibase/budibase/issues/10445. Feel free to add an additional information to the comments.

There may be a potential workaround, since you are using a webhook you could use the rest data source and then create a custom query which could be run using the `Execute Query` action. These I believe will have the responses available to you to use in the way you described

I hope this helps!

**Reply by manwithpowers** [reply]
Cheers, thanks. I was thinking of using an Execute Query action as a work around myself and that should work for the time being.
